src/servers/main.cc

```diff
@@ -1150,4 +1151,12 @@ main(int argc, char** argv)
   if (keep_alive == nullptr) {
     return 1;
   }
+
+  // Return error code if failed to stop gracefully.
+  if (stop_err != nullptr) {
+    TRTSERVER_ErrorDelete(stop_err);
```

Comment:
macro `LOG_IF_ERR` already invokes `TRTSERVER_ErrorDelete`, exiting directly should be fine.

Reply:
Yes, shouldn't use stop_err after that point since it is deleted. Fixed.